packages/backend/src/services/searchService/integrationTests/utilities/requestBuilder/buildRegexp.js

```diff
@@ -1,3 +1,120 @@
-module.exports = function buildRegexp(value) {
-  return value
+const sanitizeInput = input => {
+  const withoutDoubleBlanks = input.replace(/\s\s+/g, ' ')
+  return withoutDoubleBlanks
+}
+
+const extractFlags = input => {
+  let hasSpace = false
+  let hasEqual = false
+  let hasStar = false
+  let hasPhrase = false
+  if (input.includes(' ')) {
+    hasSpace = true
+  }
+
+  if (input.includes('=')) {
+    hasEqual = true
+  }
+
+  if (input.includes('*')) {
+    hasStar = true
+  }
+
+  if (input.includes('"')) {
```

Comment:
do you want to check for an even number of `"` (greater than 0)? (considering what Markus said today)

Reply:
I wait for further instructions regarding this. In later commits I added test of the input that throws error if the format is not correct. I prefer to keep it like this for now and then potentially make the validation less demanding later